redbeams-api/src/main/java/com/sequenceiq/redbeams/api/endpoint/v4/database/request/DatabaseRequestParameters.java

```diff
@@ -0,0 +1,27 @@
+package com.sequenceiq.redbeams.api.endpoint.v4.database.request;
+
+import static com.sequenceiq.redbeams.doc.ModelDescriptions.Database;
+
+import java.io.Serializable;
+
+import javax.validation.constraints.NotNull;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+import io.swagger.annotations.ApiModelProperty;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class DatabaseRequestParameters implements Serializable {
```

Comment:
Do we include serialVersionUID on Serializable classes?

Reply:
Good idea. However I see it is not really used in cloudbreak. There are some SerialVersionUIDs in the project, but those are I guess copied from Thunderhead or some other cloudera sources related to UMS.